tests/unit-tests/test_evaluator.py

```diff
@@ -197,23 +172,22 @@ def test_mrr_per_etype_lp_evaluation():
 
     th.distributed.destroy_process_group()
 
-def test_mrr_lp_evaluator():
+def test_lp_evaluator():
```

Comment:
Let's keep the name of test_mrr_lp_evaluator. We will implement hit@ later.

Reply:
changed.